src/molify/packmol.py

```
@@ -0,0 +1,102 @@
+        Combined stdout and stderr from packmol.
+
+    Raises
+    ------
+    RuntimeError
+        If packmol execution fails.
+    FileNotFoundError
+        If the input file doesn't exist.
+    """
+    input_path = Path(input_file)
+    if not input_path.exists():
+        raise FileNotFoundError(f"Input file not found: {input_path}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)